src/nile/core/account.py

```diff
@@ -65,7 +78,10 @@ def deploy(self):
 
     def send(self, to, method, calldata, max_fee, nonce=None):
         """Execute a tx going through an Account contract."""
-        target_address, _ = next(deployments.load(to, self.network)) or to
+        target_address, _ = next(deployments.load(hex_address(to), self.network)) or to
+        # Work with integers internally
+        target_address = normalize_number(target_address)
```

Comment:
```suggestion
        target_address = hex(target_address)
```
`normalize_number` gave me a lot of trouble here. I think using `hex` is the way to go

Reply:
What troubles did you find? Using hex here is against the pattern we agreed, we should use int internally. Anyway, normalize_number is not necessary here because deployments.load always returns an int (being compliant with the pattern). I removed it.

Reply:
I believe the issues were from how I was hacking the `deployments.load` when trying to get it to work locally. With the latest updates to the PR, everything's running well